c2corg_ui/static/js/documentediting.js

```diff
@@ -551,10 +589,23 @@ app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
     // init empty conditions_levels for ng-repeat
     outing['locales'][0]['conditions_levels'] = [{'level_snow_height_soft': '', 'level_snow_height_total': '', 'level_comment': '', 'level_place': ''}];
   }
-
   return outing;
 }
 
+/**
+ * @returns {Object} url parameters
+ * @private
+ */
+app.DocumentEditingController.prototype.parseUrl_ = function() {
```

Comment:
$location works as good as the function i had created, but now changes of URL don't redirect anymore. Clicking on the menu changes the url, but does nothing. It works on the other pages where we don't use $location.